buildSrc/src/main/java/org/elasticsearch/gradle/testfixtures/TestFixturesPlugin.java

```diff
@@ -95,9 +95,15 @@ public void apply(Project project) {
                 (name, port) -> postProcessFixture.getExtensions()
                     .getByType(ExtraPropertiesExtension.class).set(name, port)
             );
+
+            // Make it possible to have the both fixture and what's using it in a single place
+            extension.fixtures.add(project);
```

Comment:
The `TestFixtureExtension` already holds a reference to the owning project. Why muddle this collection when we can just expose that existing property?

Reply:
`extension.fixtures` are the project where the fixture is used and we want to apply  configuration too.  The  current project can be one of those, so it too needs to be configured.

Reply:
Got it. The project under test can effectively depend on itself.